jail-supybot.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: configure supybot
+  hosts: supybot
+  gather_facts: no
+  connection: local
+  sudo: yes
+  vars:
+    env_name: venv
+    web_host_name: "supybot.buildbot.net"
+    master_service_name: "supybot"
+
+  roles:
+  - base
+  - role: user
+    user_id: "{{ worker_account }}"
+    user_name: Buildbot Worker Account
+  - role: virtualenv
+    venv_user: "{{ worker_account }}"
+    venv_home_dir: "{{ getent_passwd[worker_account].4 }}"
```

Comment:
I _think_ I tried to use `~user` at some point and it just did not work.

If it works, I'd gladly get rid of `getent`.